app/src/main/res/layout/dialog.xml

```diff
@@ -44,7 +45,7 @@
       />
   <TextView
       android:id="@+id/save_tv"
-      style="@style/TextAppearance.AppCompat.Small"
+      style="@style/TextAppearance.AppCompat.Medium"
```

Comment:
what do you think about this ?
looks good, matches our functionality

https://material.io/develop/android/components/buttons/#text-button

Reply:
Looks good!